python/cuml/dask/common/base.py

```diff
@@ -51,8 +52,18 @@ def __init__(self, client=None, verbose=False, **kwargs):
     def get_combined_model(self):
         """
         Return single-GPU model for serialization
+
+        Returns
+        -------
+
+        mode : Trained single-GPU model or None if the model has not
```

Comment:
s/mode/model/

Also, it looks like this raises an exception if not fitted rather than returning None per the comment?

Reply:
I guess I lean towards returning None in that case